tools/payments/cmd/authorize/main.go

```diff
@@ -90,6 +90,10 @@ func main() {
 		log.Printf("Redis: %s, %s\n", *redisAddr, *redisUser)
 	}
 
+	if *env != "dev" && len(*pcr2) != 96 {
```

Comment:
Should `env` here and below be consistent with other places?

Reply:
When passing the `-e` flag to the tooling we've been using the abbreviated `dev`. However, this is probably a good place to enforce a better practice. Fixed.